provider/azure/environ.go

```diff
@@ -1110,11 +1123,12 @@ func (env *azureEnviron) StopInstances(ctx context.ProviderCallContext, ids ...i
 	cancelResults := make([]error, len(ids))
 	for i, id := range ids {
 		logger.Debugf("canceling deployment for instance %q", id)
+		sdkCtx := stdcontext.Background()
```

Comment:
This needs to be done inside the go func below, just after the defer wg.Done() line.